snow/networking/router/chain_router_test.go

```diff
@@ -711,12 +725,15 @@ func TestRouterHonorsRequestedEngine(t *testing.T) {
 		"",
 		prometheus.NewRegistry(),
 	))
+	defer chainRouter.Shutdown(context.TODO())
```

Comment:
We should use `context.Background` in tests instead of `TODO`

Reply:
Done

Reply:
Done